Artesian/Artesian.SDK/Dto/UnitOfMeasure/CheckConversionResults.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections.Generic;
+
+namespace Artesian.SDK.Dto.UnitOfMeasure
+{
+    /// <summary>
+    /// The CheckConversionResults class.
+    ///             TargetUnitOfMeasure = the target unit of measure
+    ///             ConvertibleInputUnitOfMeasure = list of convertible input unit of measure
+    ///             NotConvertibleInputUnitOfMeasure = list of not convertible input unit of measure
+    /// </summary>
+    public record CheckConversionResults
+    {
+        /// <summary>
+        /// CheckConversionResults Constructor
+        /// </summary>
+        public CheckConversionResults()
+        {
+            _init();
+        }
+
+        /// <summary>
+        /// CheckConversionResults Constructor
+        /// </summary>
+        /// <param name="targetUnitOfMeasure"></param>
+        public CheckConversionResults(string targetUnitOfMeasure)
+        {
+            TargetUnitOfMeasure = targetUnitOfMeasure;
+            _init();
+        }
+
+        /// <summary>
+        /// Initializes the CheckConversionResults object
+        /// </summary>
+        private void _init()
```

Comment:
why not just newing them in the property definition?

public X {get;set;} = new X()